client/src/pages/dashboard/Dashboard.js

```diff
@@ -126,6 +125,8 @@ function Dashboard() {
                                 changeDescription={(x) => {setProposalDescription(x)}}
                                 title={proposalListDefault[i].title} 
                                 description={proposalListDefault[i].description}
+                                displayX="false"
```

Comment:
again here, changing the name of displayX to be more accurate would be good! For instance, deleteMode or editMode or manageMode etc